### PR TITLE
Feature/fix broken link

### PR DESCRIPTION
**Pull request by krebil**
## Description

_What did you add/update/change?_

Fixed a link to the property expansion and limiting article. From https://docs.umbraco.com/umbraco-cms/reference/content-delivery-api#property-expansion-and-limiting under concepts

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [x] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

File: 14/umbraco-deploy/getting-started/deploy-settings.md [hunk 1]
```diff
@@ -56,7 +56,6 @@ For illustration purposes, the following structure represents the full set of op
             "TransferDictionaryAsContent": false,
             "IgnoreMissingLanguagesForDictionaryItems": false,
             "SetEmptyDictionaryItemsOnTransfer": true,
-            "IgnoreMissingLanguagesForDictionaryItems": false,
```
Comment (sofietoft) [inline 14/umbraco-deploy/getting-started/deploy-settings.md:59]: In your description, you do not mention removing this? 🤔 
Perhaps it's an error?

**Review by sofietoft — COMMENTED** [review]
Good catch @krebil ! 😄 

I've left a comment - take a look when you have a minute!

File: 14/umbraco-deploy/getting-started/deploy-settings.md (hunk 1 above)
Comment (krebil) [reply 14/umbraco-deploy/getting-started/deploy-settings.md:59]: Must be a leftover from a previous commit, I'll remove asap

**Comment by krebil** [comment]
@sofietoft Should be fixed now 😊

**Review by sofietoft — APPROVED** [review]
Ah, I thought as much! Thanks for taking care of it 💪 

And thanks again for the PR! 😄 
I'll get it merged right away.

**Pull request merged**